src/main/java/edu/ucsb/mapache/repositories/SlackUserRepository.java

```diff
@@ -12,4 +12,7 @@
 public interface SlackUserRepository extends MongoRepository<SlackUser, ObjectId> {
     @Query("{ 'profile.email': ?0}")
     List<SlackUser> findByEmail(String email);
+
+    @Query("{ 'is_admin': true")
```

Comment:
This search query doesn't have a matching closing brace. Can we fix that?

Reply:
Done